stanford_migrate.module

```diff
@@ -321,3 +323,84 @@ function stanford_migrate_execute_migration(MigrationInterface $migration, strin
 function stanford_migrate_migration_list() {
   return \Drupal::service('stanford_migrate')->getMigrationList();
 }
+
+/**
+ * Implements hook_form_alter().
+ */
+function stanford_migrate_form_alter(&$form, $form_state, $form_id) {
+  if ($form_id == 'media_image_edit_form') {
+    // If a media entity was imported via migration, disable removal.
+    $media = \Drupal::routeMatch()->getParameter('media');
+    if ($media && $media instanceof Media) {
+      $mediaId = $media->id();
+
+      /*
+       * Contains content type as the top-level key, and fields on each content
+       * type to disable removal of images.
+       */
+      $fieldsToDisable = [
+        'hs_person' => [
+          'field_hs_person_square_img',
+          'field_hs_person_image',
+        ],
+        'hs_event' => [
+          'field_hs_event_image',
+        ],
+        'hs_news' => [
+          'field_hs_news_image',
+        ],
+        'hs_publications' => [
+          'field_hs_publication_image',
+        ],
+      ];
```

Comment:
@codechefmarc Could we put this in a configuration setting for the module and then load it from the configuration? I know we have these documented in the ADR as is so that may be enough for now, but it would be nice to have these stored in configuration if possible.

Reply:
I added to the config and in the main repo, added a config so it should pick it up to use.